packages/components/src/List/List.component.js

```diff
@@ -165,7 +168,7 @@ function List({
 				components={components}
 			/>
 			{injected('after-toolbar')}
-			{selectAllCheckbox && <SelectAll {...selectAllCheckbox} />}
+			{selectAllCheckbox && <SelectAll {...selectAllCheckbox} t={t} />}
```

Comment:
What do you think of having the `SelectAll` component make the useTranslation by himself ?